components/custom-radio-button/css/stories/states.stories.mdx

```diff
@@ -0,0 +1,46 @@
+<!-- @license CC0-1.0 -->
+
+import { Canvas, Meta, Story } from "@storybook/addon-docs";
+import { CustomRadioButton, defaultArgs } from "../template";
+import "../index.scss";
+
+<Meta
+  id="css-custom-radio-button"
+  title="CSS Component/Custom Radio Button/States"
+  argTypes={{}}
+  parameters={{
+    status: {
+      type: "WORK IN PROGRESS",
+    },
+  }}
+/>
+
+# CustomRadio Button Component states
```

Comment:
We don't use it consistently yet, but for new Storybook titles and headings we intend to use sentence casing, for example: `Custom radio button component states`